package.json

```diff
@@ -83,16 +84,15 @@
   "config": {
     "forge": {
       "packagerConfig": {
-        "asar": true
+        "asar": true,
+        "icon": "./easybloqs-icon.ico"
       },
       "makers": [
         {
           "name": "@electron-forge/maker-squirrel",
           "config": {
-            "name": "easybloqs",
             "setupExe": "EasyBloqsSetup.exe",
```

Comment:
Oh, forgot this one. The default is `${appName}-${packageJSON.version} Setup.exe` apparently. That looks suitable?

Reply:
I guess it does! I will remove this line.